app/routes/score.js

```diff
@@ -1,35 +1,104 @@
+const senders = require('../messaging/senders')
+const createMsg = require('../messaging/create-msg')
+const Wreck = require('@hapi/wreck')
+const questionBank = require('../config/question-bank')
+const pollingConfig = require('../config/polling')
+function createModel (errorMessage, errorSummary, data) {
   return {
     backLink: '/collaboration',
-    inputScore: {
-      id: 'score',
-      name: 'score',
-      classes: 'govuk-input--width-10',
-      label: {
-        text: '[Placeholder] - /score',
-        classes: 'govuk-label--l',
-        isPageHeading: true
-      },
-      hint: {
-        html: 'Page yet to be built'
+    ...errorSummary ? { errorList: errorSummary } : {},
+    ...data,
+    ...(errorMessage ? { errorMessage: { text: errorMessage } } : {}),
+    nextlink: '/business-details'
+  }
+}
+
+async function getResult (correlationId) {
+  const url = `${pollingConfig.host}/desirability-score?correlationId=${correlationId}`
+
+  for (let i = 0; i < pollingConfig.retries; i++) {
+    await new Promise(resolve => setTimeout(resolve, pollingConfig.interval))
+    try {
+      const response = await Wreck.get(url, { json: true })
+
+      switch (response.res.statusCode) {
+        case 202:
+          console.log('202 received, backend didn\'t have result, continue polling')
+          break
+        case 200:
+          console.log('200 received, got result from backend, stop polling')
+          return response.payload
+        default:
+          console.log('Unhandled status code, stop polling')
+          return null
       }
+    } catch (err) {
+      // 4xx and 5xx errors will be caught here along with failure to connect
+      console.log(`${err}`)
+      return null
     }
   }
+
+  console.log(`Tried getting score ${pollingConfig.retries} times, giving up`)
+  return null
 }
 
-module.exports = [
-  {
-    method: 'GET',
-    path: '/score',
-    handler: (request, h) => {
-      return h.view('score', createModel())
-    }
-  },
-  {
-    method: 'POST',
-    path: '/score',
-    handler: (request, h) => {
-      return h.redirect('./business-details')
+module.exports = [{
+  method: 'GET',
+  path: '/score',
+  handler: async (request, h) => {
+    try {
+      // Always re-calculate our score before rendering this page
+      await senders.sendProjectDetails(createMsg.getDesirabilityAnswers(request), request.yar.id)
+
+      // Poll for backend for results from scoring algorithm
+      // If msgData is null then 500 page will be triggered when trying to access object below
+      const msgData = await getResult(request.yar.id)
+      if (msgData) {
+        const questions = msgData.desirability.questions.map(question => {
+          const bankQuestion = questionBank.questions.filter(x => x.key === question.key)[0]
+          question.title = bankQuestion.title
+          question.desc = bankQuestion.desc ?? ''
+          question.url = bankQuestion.url
+          question.unit = questionBank?.unit
```

Comment:
`questionBank?.unit` -> `bankQuestion?.unit`

Currently the unit isn't being shown